templates/node/node.py

```diff
@@ -56,9 +64,11 @@ def _get_proxy(self, template_uid, service_name):
         :param service_name: name of the service
         """
 
-        matches = self.api.services.find(template_uid=template_uid, name=service_name)
+        matches = self.api.services.find(
```

Comment:
if you're expecting one result otherwise you want to raise an error, why aren't you using `services.get`? it will raise an error if more/less than 1 service is found.